backend/src/nodes/nodes/image_filter/sharpen.py

```diff
@@ -21,7 +21,7 @@ def __init__(self):
         ]
         self.outputs = [ImageOutput(image_type="Input0")]
         self.category = ImageFilterCategory
-        self.name = "Sharpen"
+        self.name = "Sharpen (Unsharp)"
```

Comment:
Unless it's too big to fit in the width of the node selector thingy, i would use "unsharp mask" for it.

Reply:
That's how I had it originally, but it ended up leaving a gap to the right of the NumberInput it was so large.

Reply:
What about having separate subcategories for blur and sharpen so this can just be called "Unsharp Mask" and the new one can just be called "Sharpen"

Reply:
Sounds good, mostly. I'd rather be more specific naming the new one though. Calling it Sharpen when we currently have Unsharp Mask called Sharpen would be confusing.

Reply:
Tbh the way you have it now is fine